### ISSUE TITLE
Documentation nav is broken

### ISSUE THREAD

**Issue by emmills3** [foundation/foundation-sites#12578]
## Expected Behavior
Navigation it documentation is broken

...

## Current Behavior
Links go to the root, rather than the /sites/docs/ subdirectory.

For example, clicking General > XY Grid goes to:
https://get.foundation/xy-grid

But it should direct here:
 https://get.foundation/sites/docs/xy-grid

**Comment by joeworkman** [comment]
Discussion going on related to this... https://github.com/foundation/foundation-sites/discussions/12565

I will pick this issue to be the "official" issue to track.

**Comment by joeworkman** [comment]
All fixed.... 

OMG! So it's all fixed now. But let me tell you the story of the past 12 hours.... 

The build has worked and had not changed for like 7 months. However, the links were broken on server hosted by Netlify. Foundation required a pretty old version of node. Netlify suggested that I update node. Well in order to do that, I needed to completely overhaul the dependencies and run a million tests to ensure that we could run on a newer version of node. 

So win No. 1 today is that we have now have [Foundation 6.8.0](https://github.com/foundation/foundation-sites/releases/tag/v6.8.0). This version now requires node 16 but will also run great under node 18 and 20. This is a pretty awesome win if you ask me. 

With all of that published, I was feeling pretty good with myself and published the new build to Netlify. No dice. 😭 

After probing and looking around, Pretty URLs was enabled on the site. This completely modifies your URLs to your pages. I turned off that checkmark and it was magically fixed! They must have recently launched this feature and turned it on by default. 

It could have been a 30sec fix had I known. But now we have F6.8, so let's call it a good day. 😎
